pkg/scheduler/internal/queue/events.go

```diff
@@ -70,3 +74,28 @@ const (
 	// NodeConditionChange is the event when node condition is changed.
 	NodeConditionChange = "NodeConditionChange"
 )
+
+var clusterEventReg = map[string]framework.ClusterEvent{
+	AssignedPodAdd:              {Resource: framework.Pod, ActionType: framework.Add},
+	AssignedPodUpdate:           {Resource: framework.Pod, ActionType: framework.Update},
+	AssignedPodDelete:           {Resource: framework.Pod, ActionType: framework.Delete},
+	NodeAdd:                     {Resource: framework.Node, ActionType: framework.Add},
+	NodeSpecUnschedulableChange: {Resource: framework.Node, ActionType: framework.UpdateNodeUnschedulable},
+	NodeAllocatableChange:       {Resource: framework.Node, ActionType: framework.UpdateNodeAllocatable},
+	NodeLabelChange:             {Resource: framework.Node, ActionType: framework.UpdateNodeLabel},
+	NodeTaintChange:             {Resource: framework.Node, ActionType: framework.UpdateNodeTaint},
+	NodeConditionChange:         {Resource: framework.Node, ActionType: framework.UpdateNodeCondition},
+	PvAdd:                       {Resource: framework.PersistentVolume, ActionType: framework.Add},
+	PvUpdate:                    {Resource: framework.PersistentVolume, ActionType: framework.Update},
+	PvcAdd:                      {Resource: framework.PersistentVolumeClaim, ActionType: framework.Add},
+	PvcUpdate:                   {Resource: framework.PersistentVolumeClaim, ActionType: framework.Update},
+	StorageClassAdd:             {Resource: framework.StorageClass, ActionType: framework.Add},
+	CSINodeAdd:                  {Resource: framework.CSINode, ActionType: framework.Add},
+	CSINodeUpdate:               {Resource: framework.CSINode, ActionType: framework.Update},
+	ServiceAdd:                  {Resource: framework.Service, ActionType: framework.Add},
+	ServiceUpdate:               {Resource: framework.Service, ActionType: framework.Update},
+	ServiceDelete:               {Resource: framework.Service, ActionType: framework.Delete},
+	UnschedulableTimeout:        framework.WildCardEvent,
+	// To-be-discussed: do we really need the Unknown event?
+	Unknown: framework.WildCardEvent,
```

Comment:
We control all the events there are; so, no.

Reply:
If so, I'd like to remove:

https://github.com/kubernetes/kubernetes/blob/07bd985724efcb8e2152541f59a6a9bf398fc8b3/pkg/scheduler/eventhandlers.go#L126-L127

WDYT?

Reply:
Isn't that a noop? If so, yes, let's remove it.

Reply:
It's a noop IMO. Created #98899 to eliminate it.